content/applications/websites/website/configuration/google_search_console.rst

```diff
@@ -7,21 +7,165 @@ monitor, maintain, and troubleshoot their site's presence in Google Search resul
+=======================================
+
+.. _GSC-DNS-Record:
+
+DNS Record
+----------
+
+With this method, you add a DNS TXT record to your domain's DNS settings. Google provides you with a
+unique string to add as a TXT record. Once added, Google verifies ownership by checking the DNS
+settings.
+Start by copying the text record from the dialog google-site-verification=123aBc… after entering
+your domain.
+After you change your DNS record, go back to Search Console and click Verify in the verification pane.
+Once again, if the verification does not work immediately, wait a few hours or a day and try again,
+as the change may take some time to take effect. Just choose the unverified property from the
+:guilabel:`Properties` drop-down and it will try to verify you automatically.
+
+.. important:: To stay verified, do not remove the DNS Record even after verification succeeds.
+
+.. _GSC-HTML-File-Upload:
+
+HTML File Upload
+----------------
+
+This method involves uploading an HTML file provided by Google to the root directory of your website.
+Google verifies ownership by checking for this file. You will typically download the HTML verification
+file from Google Search Console, and then upload it to your website using FTP or file manager provided
+by your web hosting service. Odoo is the website builder that allows web hosting service on
+:doc:`/administration/odoo_online` and :doc:`Odoo.sh </administration/odoo_sh/overview/introduction>`.
+If you are using Odoo Online, it is a :abbr:`SaaS (Software as a service)` solution, meaning that it
+is not :abbr:`FTP (File Transfert Protocol)`. Odoo Online provides private databases which are fully
+managed and hosted by Odoo. Here, you just have to copy-paste the code in the appropriate field in
+the Website Settings. You will need to have permission to upload the file to your website where it
+can be accessed by a browser. Once you added your website URL under the URL prefix option and clicked
+on :guilabel:`continue`, expand the HTML file section where you will find a file :guilabel:`Download`
+button.
+
+.. image:: google_search_console/html-file-download.png
+   :alt: HTML file download
+
+Download your HTML verification file and upload it to the root directory of the website you want to
+verify.
+
+.. image:: google_search_console/open-copy-html-file.png
+   :alt: Open and copy html file
+
+Then, access your Odoo database, go to :menuselection:`Website --> Configuration --> Settings`,
+and enable :guilabel:`Google Search Console` in the :guilabel:`SEO` section. Paste the google123abc.html
+code in the dedicated field.
+
+.. image:: google_search_console/paste-html-code-settings.png
+   :alt: Paste html code in Odoo
+
+Once you complete this step, click :guilabel:`Verify` in Search Console. If you perform the steps
+above correctly, verification should work immediately.
+
+.. example:: If you want to verify www.example.com, then the root directory is the homepage. If you
```

Comment:
For this example, I recommend you to remove. After reading it, it is unnecessary to explain about /blog/ path because the Google Search Console option in Odoo verifies at the path /google<string(length=16):key>.html
It's not at /blog/google<string(length=16):key>.html
https://github.com/odoo/odoo/blob/dcd426df41db9200b95d04ba664441053281db78/addons/website/controllers/main.py#L784
Customers look to verify *domain* using site verification not the *domain's path* using site verification.

Reply:
Done! Thank you for the explanation :)